codegen/src/lib.rs

```diff
@@ -15,6 +15,38 @@
 // along with subxt.  If not, see <http://www.gnu.org/licenses/>.
 
 //! Library to generate an API for a Substrate runtime from its metadata.
+//!
+//! ## Generated Structure
+//!
+//! The API generator logic:
+//! - At the root there is the `item_mod` provided (ie `pub mod api {}`)
+//! - Pallet are represented by a child module (ie `pub mod PalletName {}`) of the root
```

Comment:
Is this -> "A pallet is ..." or -> "Pallets are ..."

Reply:
Yep, its `pallets`